src/app/search/directives/search-story.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DatePipe } from '@angular/common';
+import { ROUTER_DIRECTIVES } from '@angular/router';
+
+import { HalDoc } from '../../shared/cms/haldoc';
+import { DurationPipe, ImageLoaderComponent, OverflowComponent, StoryModel } from '../../shared';
+
+@Component({
+  directives: [ImageLoaderComponent, OverflowComponent, ROUTER_DIRECTIVES],
+  pipes: [DatePipe, DurationPipe],
+  selector: 'publish-search-story',
+  styleUrls: ['search-story.component.css'],
+  template: `
+    <div class="media-container">
+      <section class="story-link">
+        <a [routerLink]="editStoryLink">
+          <image-loader [src]="storyImage" [imageDoc]="storyImageDoc"></image-loader>
+        </a>
+        <h2>
+          <a *ngIf="storyTitle" [routerLink]="editStoryLink">{{storyTitle}}</a>
+          <a *ngIf="!storyTitle" [routerLink]="editStoryLink">(Untitled)</a>
+        </h2>
+        <p class="duration">{{storyDuration | duration}}</p>
+        <p class="modified">{{storyUpdated | date:"MM/dd/yy"}}</p>
+        <p *ngIf="storyAudioTotal" class="audio-total"><i class="icon-up-dir"></i>{{storyAudioTotal}}</p>
+        <p *ngIf="statusClass" [class]="statusClass">{{statusText}}</p>
+      </section>
+      <section class="story-detail">
+        <a *ngIf="seriesLink" [routerLink]="seriesLink">{{seriesTitle}}</a>
+        <h3 *ngIf="!seriesLink">{{seriesTitle}}</h3>
+        <overflow [overflowText]="storyDescription"></overflow>
+        <section class="story-tags">
+          <span *ngFor="let tag of storyTags">{{tag}}</span>          
+        </section>
+      </section>
+    </div>
+  `
+})
+
+export class SearchStoryComponent implements OnInit {
+
+  @Input() story: StoryModel;
+
+  editStoryLink: any[];
+  seriesLink: any[];
+  isEmptyDraft: boolean;
+
+  storyId: number;
+  storyImage: string;
+  storyImageDoc: HalDoc;
+  storyTitle: string;
+  storyDuration: number;
+  storyAudioTotal: number;
+  storyUpdated: Date;
+  storyDescription: string;
+  storyTags: string[];
+  seriesTitle: string;
+
+  statusClass: string;
+  statusText: string;
+
+  ngOnInit() {
+    this.storyId = this.story.id;
+    this.isEmptyDraft = this.story.isNew && !this.story.isStored();
+    this.storyTitle = this.story.title;
+    this.storyUpdated = this.story.lastStored || this.story.updatedAt;
+    this.storyDescription = this.story.shortDescription;
+    this.storyTags = this.story.extraTags && this.story.extraTags.length > 0 ? this.story.extraTags.split(', ') : [];
+
+    if (this.story.isNew) {
```

Comment:
I was thinking that "new" stories (only saved in localstorage) wouldn't be included on this page.  Just to avoid the trouble of attempting to injecting them into the persisted CMS results.

So the only place a "new" story would be accessible, would be on the Home page, where they're always listed before the saved stories in the series.

I think.